crypto/data_formats.go

```diff
@@ -42,9 +42,9 @@ func HashLogRoot(root trillian.SignedLogRoot) []byte {
 	// JSON encoded otherwise (it uses floats). We want to be sure that people
 	// using JSON to verify hashes can build the exact same input to ObjectHash.
 	rootMap := map[string]string{
-		mapKeyRootHash:       base64.StdEncoding.EncodeToString(root.RootHash),
-		mapKeyTimestampNanos: strconv.FormatInt(root.TimestampNanos, 10),
-		mapKeyTreeSize:       strconv.FormatInt(root.TreeSize, 10)}
+		mapKeyRootHash:       base64.StdEncoding.EncodeToString(root.GetRootHash()),
```

Comment:
root is not a pointer, you can't really pass nil to this function.  (Maybe it *should* be a pointer, but that's another discussion.)